docs/source/faq.rst

```diff
@@ -205,7 +205,36 @@ Getting info about environments
 
 SEE ALSO: the :doc:`/using/envs` section of our documentation site. 
 
-2. How can I list all installed packages in a specific environment, for example, ``myenv``?
+2. How can I show the name of the active environment in my command prompt?
+
+   The anaconda (or miniconda) installer asks whether it should always add the name 
+   of the active environment to the command prompt by inserting it into the environment 
+   variable called PATH in the Bash shell. The default is no.
+
+   To turn it on later:
+   
+     .. code-block:: bash
+        
+        source deactivate
```

Comment:
When I set changeps1 false while in an environment, the name of the environment that was current then remained in my prompt as I switched around between different environments. When I set changeps1 true, it didn't change my prompt until after the next source activate or source deactivate command. This behavior does make perfect sense, but if we assume that users want to see no environment name whenever changeps1 is false, and the correct environment name whenever changeps1 is true, then they should run 'source deactivate' before changing changeps1.